packages/plugin-mcp/src/mcp/tools/resource/delete.ts

```diff
@@ -78,13 +83,15 @@ export const deleteResourceTool = (
         collection: collectionSlug,
         depth,
         user,
+        ...(locale && { locale }),
+        ...(fallbackLocale && { fallbackLocale }),
       }
 
       // Delete by ID or where clause
-      if (id) {
-        deleteOptions.id = id
+      if (idString) {
+        deleteOptions.id = idString
```

Comment:
Why are we doing this? Won't this break PG? passing a string ID into the payload.delete call? Im confused why we are converting the id to a string really at all.

Reply:
Hi @JarrodMFlesch 

This is to address the following issue:

PostgreSQL ID Compatibility (e004c551d)

Issue: 4 localization tests failing in PostgreSQL due to numeric vs string ID mismatch
Fix: Updated MCP tool schemas to accept both string | number for IDs, with conversion to string for Payload operations
Files: Updated schemas and resource handlers (find/update/delete)